pulitzer.gemspec

```diff
@@ -32,6 +32,9 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'sqlite3', '~> 1.3'
   s.add_development_dependency 'minitest', '~> 5.1'
   s.add_development_dependency 'byebug', '~> 8.0'
+  s.add_development_dependency 'pry-rails', '~> 0.3', '< 1.0'
```

Comment:
why adding pry when we already have byebug? is there something that pry can do and byebug don't?